### ISSUE TITLE
Backup failed with Pagination Error

### ISSUE THREAD

**Issue by shabin5785** [alexdebrie/serverless-dynamodb-backups#5]
Got the below error from the lambda:

"Error removing stale backups. Error: Operation cannot be paginated: list_backups".
My configuration is correct as the tables i have provided in tables.json is listed and i get a slack alert as the backup has been successful. But the backup files are missing..

**Comment by nottakis** [comment]
I had the same issue so I created the easy fix. https://github.com/alexdebrie/serverless-dynamodb-backups/pull/6 

Also we need to fix the iamRole like this. 
I didn't add this code to the pull-request because we need to be cafeful:)
```
iamRoleStatements:
    - Effect: "Allow"
      Action:
        - "dynamodb:CreateBackup"
      Resource:
        Fn::Join:
          - ":"
          - - "arn:aws:dynamodb"
            - Ref: "AWS::Region"
            - Ref: "AWS::AccountId"
            - "table/*"
    - Effect: "Allow"
      Action:
        - "dynamodb:ListTables"
      Resource: "*"
    - Effect: "Allow"
      Action:
        - "dynamodb:ListBackups"
      Resource: "*"
    - Effect: "Allow"
      Action:
        - "dynamodb:DeleteBackup"
      Resource: "*"
```

**Comment by alexdebrie** [comment]
Closed by #6
